pallets/randomness/src/inherent.rs

```diff
@@ -1,14 +1,14 @@
 use codec::Encode;
 use sp_inherents::{InherentIdentifier, IsFatalError};
-use sp_runtime::RuntimeString;
+extern crate alloc;
 
 /// BABE VRF Inherent Identifier
 pub const INHERENT_IDENTIFIER: InherentIdentifier = *b"babe_vrf";
 
 #[derive(Encode)]
 #[cfg_attr(feature = "std", derive(Debug, codec::Decode))]
 pub enum InherentError {
-    Other(RuntimeString),
+    Other(alloc::borrow::Cow<'static, str>),
```

Comment:
Maybe? https://crates.parity.io/sp_std/borrow/enum.Cow.html

Reply:
This is reimport of alloc::borrow::Cow.